src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -184,23 +222,25 @@ const NavHeader = (props) => {
           />
         </View>
       )}
-    </Animated.View>
+    </View>
   );
 };
 
 NavHeader.propTypes = {
+  initialData: PropTypes.object,
   onShareCommunity: PropTypes.func,
   isHeaderHide: PropTypes.bool,
   isFollow: PropTypes.bool,
   hideSeeMember: PropTypes.bool,
   animatedHeight: PropTypes.number,
   opacityHeaderAnimation: PropTypes.number,
+  opacityImage: PropTypes.number,
   getTopicDetail: PropTypes.func,
   setMemberCount: PropTypes.func,
   memberCount: PropTypes.number,
   topicDetail: PropTypes.object,
   setIsFollow: PropTypes.func,
-
+  isLoading: PropTypes.bool,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes have been updated to reflect the new and removed props. This is good for documenting the expected types of the props. However, please ensure that all components using `NavHeader` have been updated to pass the correct props.



<!-- This is an auto-generated comment by OSS CodeRabbit -->